pkg/cli/rollback.go

```diff
@@ -20,17 +20,17 @@ import (
 	"github.com/spf13/cobra"
 )
 
-func getRollbackNewCommand() *cobra.Command {
+func getRollbackCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "rollback-new <changeId>",
 		Short: "Rolls-back a new network change",
```

Comment:
remove `new` from the `Use` and `Short`.

Reply:
done